cabal-bundler/src/CabalBundler/Main.hs

```diff
@@ -100,6 +102,7 @@ optsP = Opts
     <*> optional (O.option (O.eitherReader $ return . fromFilePath) (O.short 'p' <> O.long "plan" <> O.metavar "PATH" <> O.help "Use plan.json provided"))
 
 formatP :: O.Parser Format
-formatP = nixSingle <|> curl <|> pure Curl where
+formatP = nixSingle <|> curl <|> openbsd <|> pure Curl where
     nixSingle = O.flag' NixSingle $ O.long "nix-single" <> O.help "Single nix derivation"
     curl      = O.flag' Curl      $ O.long "curl"       <> O.help "Curl script to download dependencies"
+    openbsd   = O.flag' OpenBSD   $ O.long "openbsd"    <> O.help "OpenBSD port manifest"
```

Comment:
Is 

```
	Cabal	3.0.0.0	0	\
	ansi-terminal	0.10.1	0	\
	ansi-wl-pprint	0.6.9	2	\
	cabal-fmt	0.1.1	0	\
	colour	2.3.5	0	\
	optparse-applicative	0.15.1.0	1	\
	transformers-compat	0.6.5	0	\
```

the **whole** port manifest file, or is it just a fragment?

Reply:
Relatedly, is there a page explaining what "port manifest" files look like. I'd like to add acomment to the top of the `OpenBSD` module, but I have hard time finding any information.

(Don't worry too much about that, it's e.g. impossible to find anything concrete about nix derivations as well).

Reply:
This is a fragment of a port Makefile, the value for MODCABAL_MANIFEST variable. 

I will eventually have a stable reference for where this is documented, but I don't have a predictable location yet.

Reply:
Thanks, I amended the `--help` message to mention `MODCABAL_MANIFEST` variable.

Could you check #35 that I didn't break anything and specifically that the golden fixture looks correct